requirements/dev-bullseye-requirements.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with python 3.9
+# This file is autogenerated by pip-compile with python 3.7
```

Comment:
Similarly, I'd re-generate this file with Python 3.9 before investigating further the CI failures 🙂